### PR TITLE
ADEN-9257 Make collapsinator use forced_collapse event

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-engine/models/ad-slot.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export class AdSlot extends EventEmitter {
 
 	static STATUS_BLOCKED = 'blocked';
 	static STATUS_COLLAPSE = 'collapse';
+	static STATUS_FORCE_COLLAPSED = 'force_collapsed';
```
Comment (preludek) [inline src/ad-engine/models/ad-slot.ts]: Are everybody aware of that status name? Wasn't it discussed as 'collapsinated' or smth similar?

File: @types/ads/index.d.ts [hunk 2]
```diff
@@ -8,3 +8,5 @@ interface Runtime {
 	unblockHighlyViewableSlots?: boolean;
 	bab?: any;
 }
+
+type ManualAdType = 'collapse' | 'force_collapsed' | 'forced_success' | 'manual';
```
[the remark this answers is not in the document]
Comment (dmnsobczak) [reply @types/ads/index.d.ts]: As far as I know that's all the values we're setting in creatives with `AdEngine_adType` variable. @sseyda please let me know if I'm wrong.

File: src/ad-engine/models/ad-slot.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-engine/models/ad-slot.ts]: Yeah, but I decided that `force_collapsed` is more consistent with `force_success` status and I discussed it with @martynaut and she's fine with that.

File: @types/ads/index.d.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (sseyda) [reply @types/ads/index.d.ts]: @dmnsobczak you're right. Our collapse creative uses only `collapse` and `forced success` as values for `AdEngine_adType`

**Pull request merged**